ci/Dockerfile

```diff
@@ -62,12 +62,14 @@ RUN npm install -g vsce
 
 
 # Install CMake (for Zephyr)
+# hadolint ignore=DL4001,DL3047
 RUN wget ${WGET_ARGS} https://github.com/Kitware/CMake/releases/download/v${CMAKE_VERSION}/cmake-${CMAKE_VERSION}-Linux-${HOSTTYPE}.sh && \
 	chmod +x cmake-${CMAKE_VERSION}-Linux-${HOSTTYPE}.sh && \
 	./cmake-${CMAKE_VERSION}-Linux-${HOSTTYPE}.sh --skip-license --prefix=/usr/local && \
 	rm -f ./cmake-${CMAKE_VERSION}-Linux-${HOSTTYPE}.sh
 
 # Install Zephyr SDK
+# hadolint ignore=DL4001,DL3047
```

Comment:
Same here, with no reason, it should not be ignored. We do not want to hide the issue, we want to fix it

Reply:
DL3047 is set to be ignored, because codefactor wants the flag --progress for wget. The flag --progress is set in the WGET_ARGS therefor i set the issue to ignore. DL4001 is not set to ignore anymore, since i replaced curl with wget in order to only use wget.